Public/Src/Sandbox/MacOs/Interop/Sandbox/Data/SandboxedProcess.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "SandboxedProcess.hpp"
+#include <execinfo.h>
+#include <errno.h>
+
+static void full_write(int fd, const char *buf, size_t len)
```

Comment:
Please remove this backtrace logic, I used it to analyze the call for allocation / destruction logic.

Reply:
done